index.php

```diff
@@ -168,15 +167,13 @@
 	</div>
           </div>
     
-      <script>
+		  <script>
     function openSlideMenu(){
-      document.getElementById('side-menu').style.width = '250px';
-      document.getElementById('main').style.marginLeft = '250px';
+      document.getElementById('side-menu').style.marginLeft = '0';
     }
 
     function closeSlideMenu(){
-      document.getElementById('side-menu').style.width = '0';
-      document.getElementById('main').style.marginLeft = '0';
+      document.getElementById('side-menu').style.marginLeft = '-250px';
```

Comment:
This side navbar was very annoying to be honest, I tried several times to solve this issue but I failed...hahahaahaa..
Good job @zwnhmd , now it closes completely.